e2e/README.md

```diff
@@ -26,59 +30,68 @@ don't already have `../dev/secrets-env.sh`:
 source ../dev/secrets-env.sh
 ```
 
-Next, specify the Automate instance you wish to target.
-This could be dev or acceptance; typically start with your local box with this:
+Next, specify the Automate instance you wish to target. This could be
+dev or acceptance; typically start with your local box with this:
 
 ```bash
 export CYPRESS_BASE_URL=https://a2-dev.test
 ```
 
-If you are developing Cypress tests, you'll want to open the Cypress app
-so you can watch the browser as the tests run. The Cypress tool is great for debugging.
+If you are developing Cypress tests, you'll want to open the Cypress
+app so you can watch the browser as the tests run. The Cypress tool is
+great for debugging. Make sure you're still in the `e2e` directory of
+Automate:
 
 ```bash
 npm run cypress:open
 ```
 
-If you'd like to only get test results, you can run the tests in the terminal with Cypress in the background.
+If you'd like to only get test results, you can run the tests in the
+terminal with Cypress in the background.
 
 ```bash
 npm run cypress:run
 ```
 
-You can test against dev/acceptance environments as well.
-Just be sure to leave off the trailing `/`.
+You can test against dev/acceptance environments as well. Just be
+sure to leave off the trailing `/`.
 
 ## Running Cypress pipeline tests
 
-As part of our [deploy pipeline](https://github.com/chef/automate/blob/master/.expeditor/deploy.pipeline.yml),
-we run Cypress tests every time we deploy a new build to `dev` or `acceptance`.
-You can see latest builds on [this buildkite dashboard](https://buildkite.com/chef/chef-a2-master-deploy-dev/builds).
+As part of our [deploy
+pipeline](https://github.com/chef/automate/blob/master/.expeditor/deploy.pipeline.yml),
+we run Cypress tests every time we deploy a new build to `dev` or
+`acceptance`. You can see latest builds on [this buildkite
+dashboard](https://buildkite.com/chef/chef-automate-master-deploy-dev/builds).
 
-Your Cypress tests may be passing locally, but it's a good idea to check their stability
-when pointed at `dev` and `acceptance` environments.
+Your Cypress tests may be passing locally, but it's a good idea to
+check their stability when pointed at `dev` and `acceptance`
+environments.
 
 To do so, follow these steps.
 
-1. Log on to a docker container (using the same image buildkite uses), using your local A2 as the working directory.
+1. Log on to a docker container (using the same image buildkite uses),
```

Comment:
While it is perfectly correct to number all list items as 1, and let the md post-processor generate the sequence, I have been suggesting the alternate approach, which is also valid: using the actual sequence numbers (1, 2, 3 in this case). That allows one to read the raw markdown file with more context. And the markdownlint tool vets it equally well--it will let you know if later you put things out of order.